File: leetcode/easy/String/cpp/2027_Minimum Moves to Convert String.cpp
class Solution {
public:
    int minimumMoves(string s) {
        
        int ans = 0;
        for(int i = 0; i < s.length(); i++){
            
            if(s[i] == 'X'){
                    
                ans++;
                s.replace(i, 3, "OOO");
            }
        }
        
        return ans;
    }
};

File: leetcode/easy/greedy/2027_Minimum Moves to Convert String.java
class Solution {
    public int minimumMoves(String s) {
        
        int n = s.length();
        int ans = 0;
        
        int i = 0;
        while(i < n){
            
            if(s.charAt(i) == 'X'){
                i = i + 3;
                ans++;
            }    
            else{
                i++;
            }
            
        }
        
        return ans;
    }
}
